internal/convert.go

```diff
@@ -126,6 +126,7 @@ const (
 	ForeignKeyOnDelete
 	ForeignKeyOnUpdate
 	SequenceCreated
+	ForeignKeyActionsNotSupported
```

Comment:
nit: I think this should be ForeignKeyActionNotSupported? As in singular to be consistent with the wordings.